src/i18n/content-types.ts

```diff
@@ -53,6 +53,28 @@ interface Navbar {
 			changePassword: string;
 			deleteAccount: string;
 			approve: string;
+			usernameShould: string;
+			checkInput: string;
+			validateLength: string;
```

Comment:
I'd suggest to make it consistent with `passwordLength`, let's use `usernameLength` here  (which also makes it easier to understand imo)